lib/flying_penguin/duffel/client.ex

```diff
@@ -55,6 +35,41 @@ defmodule FlyingPenguin.Duffel.Client do
     end
```

Comment:
Is there a reason we handle 422 specifically (but not 401, 3xx or 5xx status codes, for example)?

Reply:
A lot of this is cruft from when I was debugging the response. I was getting a 422 at one point, so I wanted to process it and log it to the console. It makes sense to remove these. It probably makes more sense to only take the happy path case, and have the :error handle everything else. There is no reason that I can think of to handle anything else in a unique way.

Reply:
I agree!